src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  margin: 0 ;
+  box-sizing: border-box ;
+}
+
+.body {
+  --color-white: #fff ;
+  --color-blue: #00acdc ;
+  --color-grin: #616070 ;
+  --color-dark: #060b35 ;
+  background: #e5e5e5 ;
+  font-family: Roboto, sans-serif ;
+}
+
+.card {
```

Comment:
Create a separate file for each BEM block. Follow the checklist, please 
<img width="938" alt="image" src="https://user-images.githubusercontent.com/99403198/219865056-e68b1a45-365b-4af7-91c0-105aa52754ed.png">